<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeshViewOptionsWidget</class>
 <widget class="QDockWidget" name="MeshViewOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>210</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>210</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mesh View Options</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>191</width>
      <height>101</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>191</width>
      <height>120</height>
     </size>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>172</width>
       <height>160</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>160</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="showAxisCheckbox">
        <property name="minimumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>Show Axis</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showBBoxCheckbox">
        <property name="minimumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>Show BBox</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showParticlesCheckbox">
        <property name="minimumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="text">
         <string>Show Particles</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showFacesCheckbox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show Mesh Faces</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showEdgesCheckbox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show Mesh Edges</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showCutsCheckbox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show Cuts</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="showSurfacesCheckbox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Show Surfaces Only</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="colorByQualityCheckbox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>164</width>
          <height>21</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Color by Quality</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QSlider" name="zSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>186</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="zCheckbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>186</y>
      <width>74</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>z-axis</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QSlider" name="ySlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>166</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="clippingCheckbox">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>119</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="xSlider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>146</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Clipping:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="yCheckbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>166</y>
      <width>74</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>y-axis</string>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QCheckBox" name="xCheckbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>146</y>
      <width>74</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x-axis</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string>buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="label2_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sync:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="syncClippingCheckbox">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>119</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTreeView" name="materialsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>224</y>
      <width>191</width>
      <height>100</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>191</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>191</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label2_3">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>210</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Material Visibility Locks</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clippingCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingCheckboxClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>125</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>xCheckboxClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>yCheckboxClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>zCheckboxClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>xSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>ySliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>zSliderMoved(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zSlider</sender>
   <signal>sliderPressed()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ySlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>clippingSliderReleased()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>syncClippingCheckbox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeshViewOptionsWidget</receiver>
   <slot>syncClippingCheckboxClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>scrollingCheckboxClicked(int)</slot>
  <slot>clippingCheckboxClicked(bool)</slot>
  <slot>xCheckboxClicked(bool)</slot>
  <slot>yCheckboxClicked(bool)</slot>
  <slot>zCheckboxClicked(bool)</slot>
  <slot>xSliderMoved(int)</slot>
  <slot>ySliderMoved(int)</slot>
  <slot>zSliderMoved(int)</slot>
  <slot>clippingSliderPressed()</slot>
  <slot>clippingSliderReleased()</slot>
  <slot>syncClippingCheckboxClicked(bool)</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
